MantidPlot/src/ApplicationWindow.cpp

```diff
@@ -15175,19 +15174,19 @@ void ApplicationWindow::windowProperties() {
   s += tr("Label") + ": " + static_cast<MdiSubWindow *>(w)->windowLabel() +
        "\n\n";
 
-  if (w->isA("Matrix")) {
+  if (QString(w->metaObject()->className()) == "Matrix") {
     mbox->setIconPixmap(getQPixmap("matrix_xpm"));
     s += tr("Type") + ": " + tr("Matrix") + "\n\n";
-  } else if (w->inherits("Table")) {
+  } else if (QString(w->metaObject()->className()) == "Table") {
```

Comment:
Oh yes, this change is wrong. I'll fix/revert it. That line didn't need any fix, `inherits()` doesn't seem to be deprecated.